src/pages/create-proposal/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { HeadCell } from './proposal-types'
+import { createData } from './utils'
+
+export const rows = [
+  createData('Cupcake', 305, 3.7, 67),
+  createData('Donut', 452, 25.0, 51),
+  createData('Eclair', 262, 16.0, 24),
+  createData('Frozen yoghurt', 159, 6.0, 24),
+  createData('Gingerbread', 356, 16.0, 49),
+  createData('Honeycomb', 408, 3.2, 87),
+  createData('Ice cream sandwich', 237, 9.0, 37),
+  createData('Jelly Bean', 375, 0.0, 94),
+  createData('KitKat', 518, 26.0, 65),
+  createData('Lollipop', 392, 0.2, 98),
+  createData('Marshmallow', 318, 0, 81),
+  createData('Nougat', 360, 19.0, 9),
+  createData('Oreo', 437, 18.0, 63),
+]
+
+export const headCells: readonly HeadCell[] = [
```

Comment:
what's the point of moving these to separate file?
As I can see now they're used only in one file

Reply:
separating needing (offcourse if we can do) here to make file structure more compactable, and have different layouts of data. We need to avoid to save all structures in one place, because this make our component unreadable.